### ISSUE TITLE
Player character becomes invisible upon opening Identity menu

### ISSUE THREAD

**Issue by qwertyuiop123456373** [Draylar/identity#485]
The player character becomes invisible after opening identity menu, is persistent across worlds. The players characters hand and block breaking particles also become invisible.

List of all of my mods:
https://1drv.ms/x/s!AvwOfFl9PRJbgdkKJWjDDBmMsG6yDg?e=hGIOSB

Logs:
https://gist.github.com/qwertyuiop123456373/e6cf2dd507304bf582762177366258ae

**Comment by AlisterGontineac** [comment]
Yeah got this too. Using Magnesium and oculous, don't know if it has something to do with rendering discrepancies

**Comment by cloudythoughts06** [comment]
I have the same issue. I'm using Satin, but I don't know if it would affect it this badly.
